### ISSUE TITLE
Unusually low fps whatever I do

### ISSUE THREAD

**Issue by swardhagen** [AllTheMods/atm6-sky#117]
I am on a Macbook Pro with 16g ram (8g assigned to the game) and rarely have issues with lag while playing modded, ATM6 went just fine if I turned down the render distance a bit. To The Sky have been unusually laggy for a while and now, after about 50 hours of playtime it has become totally unplayable, with movement getting registered up to two seconds later sometimes, even with render distance set to 4, decreased particles, minimum smooth lighting and so on. I took down the dynamos, lava production and harvester and it doesn't seem to help. I can't understand what's making it like this. Anyone else having this issue? 

**To Reproduce**
1. Have a pretty decent Macbook Pro running BigSur
2. Play the game for a couple of days

**Expected behavior**
Game becomes extremely laggy with lower than 1 fps at times. 

**Screenshots**
These are my video settings:
https://ibb.co/kJp3Bbm

**Comment by celineYwY** [comment]
same thing for me exepect i have a gaming tower

**Comment by DizzyDJW** [comment]
Same. I assumed it was Mekanism Pipes rendering so I switched out all pipes. Set vid settings to lowest, can't get over 20fps anywhere. Even miles away from my base in the void. Only chunks loaded were my compact machine with storage drawers inside.

**Comment by DaRepe** [comment]
I had similar problems, 30-50 fps and constant stuttering on fairly a new skyblock base with 6900 XT, 5950x, 32gb ram, I noticed that my cpu was utilized only on 1-3 cores and was otherwise mainly idle.

I fixed it with installing optifine, now I get anywhere between 200 and 500 fps in same spot with more machines and pipes.

I read that the problem is with vanilla minecraft rendering and updating of chunks not using more than 1 cpu core, but optifine can utilize more cores, or something along those lines...

For me the rendering goes weird if I enable Fast Render and/or Render Regions in optifine.

[https://optifine.net/downloads](https://optifine.net/downloads)
